src/charm.py

```diff
@@ -224,57 +207,62 @@ def _bessd_config_file_content_matches(self, content: str) -> bool:
             return False
         return True
 
-    def _configure(self, event: EventBase) -> None:
-        """Handles events for configuring workloads.
-
-        Args:
-            event: EventBase
-        """
+    def _on_config_changed(self, event: EventBase):
+        """Handler for config changed events."""
+        if not self.unit.is_leader():
```

Comment:
This way are "non-leadr" unit ever going to be configured and have a status set for them?
In fact this applies to all `pebble_ready` handlers implemented here as well.

Reply:
No, non-leader units won't work at all and they didn't work before either. I just want to make it obvious that we don't handle scaling here.